packages/devtools_app/test/app_size_screen_test.dart

```diff
@@ -372,12 +375,12 @@ void main() {
       );
 
       appSizeController.loadTreeFromJsonFile(
-        DevToolsJsonFile(
+        jsonFile: DevToolsJsonFile(
           name: 'unsupported_file.json',
           lastModifiedTime: lastModifiedTime,
           data: unsupportedFile,
         ),
-        (error) => Notifications.of(buildContext).push(error),
+        onError: (error) => Notifications.of(buildContext).push(error),
```

Comment:
looks like this one is missing a check for mounted

Reply:
this is a callback in a test, so I left it because we can't check `if (mounted)` here